Makefile

```diff
@@ -20,24 +20,24 @@ collection-build: ## Build arista.avd collection locally.
 #########################################
 .PHONY: pyavd-build
 pyavd-build: ## Build PyAVD Python package locally.
-	cd python-avd && $(MAKE) clean build
+	cd python-avd && $(MAKE) build
 
 .PHONY: pyavd-test
 pyavd-test: ## Test PyAVD Python code with tox.
-	cd python-avd && $(MAKE) clean && tox -r
+	cd python-avd && $(MAKE) && tox -r
 
 .PHONY: pyavd-publish
 pyavd-publish: ## Build and publish PyAVD Python package.
-	cd python-avd && $(MAKE) clean build publish
+	cd python-avd && $(MAKE) build publish
 
 .PHONY: pyavd-install
 pyavd-install: pyavd-build ## Build and install PyAVD Python package.
-	pip install python-avd/dist/*
+	pip install python-avd/dist/* --force-reinstall
```

Comment:
do we want an option for `[ansible-collection]` in these commands?

Reply:
no we can re-visit later.